foundry/applications/formApplications/userManagement.d.ts

```diff
@@ -1,3 +1,70 @@
-declare class UserManagement extends FormApplication {
-  _updateObject(event?: Event, formData?: object): Promise<any>; // TODO: Add correct return type
+/**
+ * The User Management setup application
+ */
+declare class UserManagement extends FormApplication<UserManagement.Data, Users> {
+  /**
+   * @override
+   * @defaultValue
+   * ```typescript
+   * mergeObject(super.defaultOptions, {
+   *   id: "manage-players",
+   *   classes: ["dark"],
+   *   template: "templates/setup/user-management.html",
+   *   popOut: false,
+   *   closeOnSubmit: false,
+   *   scrollY: ["#player-list"]
+   * });
+   * ```
+   */
+  static get defaultOptions(): FormApplication.Options;
+
+  /** @override */
+  protected _render(force?: boolean, options?: Application.RenderOptions): Promise<void>;
+
+  /**
+   * @param options - (unused)
+   * @override
+   */
+  getData(options?: Application.RenderOptions): UserManagement.Data;
+
+  /** @override */
+  activateListeners(html: JQuery): void;
+
+  /**
+   * @param event - (unused)
+   * @override
+   */
+  protected _updateObject(event: Event, formData: Record<string, string | number>): Promise<unknown>;
```

Comment:
Can we type `formData` more concretely? Might also make sense to give the type a name.